src/data/queries/qaParagraph.ts

```diff
@@ -0,0 +1,14 @@
+import { QueryFormatter } from 'next-drupal-query'
+import { ParagraphQaParagraph } from '@/types/drupal/paragraph'
+import { QaParagraph } from '@/types/formatted/qaParagraph'
+
+export const formatter: QueryFormatter<ParagraphQaParagraph, QaParagraph> = (
+  entity: ParagraphQaParagraph
+) => {
+  return {
+    question: entity.field_question,
+    answers: entity.field_answer.map(
+      (paragraphWysiwyg) => paragraphWysiwyg.field_wysiwyg
```

Comment:
i don't think this is correct as these `field_answer` can technically be more than just wysiwyg paragraphs.

we'll likely want an approach like the one in the resourcesSupport query here: https://github.com/department-of-veterans-affairs/next-build/blob/main/src/data/queries/resourcesSupport.ts#L84